.github/workflows/integration.yml

```diff
@@ -0,0 +1,67 @@
+name: integration
+on:
+  push:
+    branches:
+      - main
+      - master
+      - '*-stable'
+  pull_request:
+jobs:
+  react-native-test-app:
+    runs-on: macos-10.15
+    if: ${{ !contains(github.event.head_commit.message, '[skip ci]') }}
+    steps:
+      - name: Set up Node.js
+        uses: actions/setup-node@v2.4.1
+        with:
+          node-version: 14
+      - name: Checkout react-native-macos
+        uses: actions/checkout@v2
+      - name: Pack react-native-macos
+        run: |
+          npm pack
```

Comment:
Adding this step prevents the build from timing out before reporting build errors. Not quite sure why though.

Reply:
Ideally, we'd do what we do in the react-native-windows repo, and complete a local publish to a verdaccio instance.  Then use that instance for the test.  -- That way you are testing even more of the complete setup.

Also these current steps will fail in a stable branch, where the version number is no longer 1000.0.0.